com.unity.template.renderstreaming-hd/Assets/Scripts/RenderStreaming.cs

```diff
@@ -49,6 +49,13 @@ public class RenderStreaming : MonoBehaviour
 
         [SerializeField, Tooltip("Array to set your own click event")]
         private ButtonClickElement[] arrayButtonClickEvent;
+
+        [SerializeField, Tooltip("Rendering target raw image about receive")]
+        private RawImage receiveImage;
```

Comment:
It is better this property is put to UIController.cs.

Reply:
I think I need new Component about receive video. (Include send offer button)
I implement `ReceiveVideoViewer.cs`.